multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
the otherway round, we don't put our template in projects template. but rather application templates.